app/lib/searching.rb

```diff
@@ -28,6 +28,7 @@ class Searching
 
   HOSTS = {
     "production" => "https://www.gov.uk/api",
+    "integration" => "https://#{ENV['USER_NAME']}:#{ENV['PASSWORD']}@www-origin.integration.publishing.service.gov.uk/api",
```

Comment:
This is a bit problematic because URLs aren't secure even if using https - they get sent in the clear and are likely to be included in log files and things.

There's an article about this here: https://www.fullcontact.com/blog/never-put-secrets-urls-query-parameters/

The short of it is you should pass an `Auth` header instead of including the credentials in the URL  (headers are encrypted for https requests). That way nothing can read them unless either end is compromised (heroku or www-origin).